src/main/java/BouquetCreator.java

```diff
@@ -0,0 +1,51 @@
+import model.Bouquet;
+import model.DecorationPlant;
+import model.Flower;
+import model.FlowerOne;
+import sun.font.Decoration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static model.FlowerColor.RED;
+import static model.FlowerColor.WHITE;
+
+public class BouquetCreator {
+
+    public Bouquet createMaxOneFlowerWRW(List<Flower> availableFlowers, List<DecorationPlant> availablDecorations) {
+
+        Bouquet bouquet = new Bouquet();
```

Comment:
Bouquet should be as simple as possible (sequential): 
create one type of flower
create another type
add all in Bouquet